views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -34,6 +34,60 @@ define([
 ], function($, __, pluginFactory, hider, previewerFactory, buttonTpl){
     'use strict';
 
+    let saved = true;
+
+    /**
+     * Handler for preview=
+     */
+    function previewHandler(e, that) {
```

Comment:
don't like name `that` can it be `plugin`?

Reply:
done